adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -142,51 +145,49 @@ - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)
-  [GADMediationAdapterPangle setCOPPA:(adConfiguration.childDirectedTreatment
-                                           ? adConfiguration.childDirectedTreatment.integerValue
-                                           : -1)];
-  _nativeRenderer = [[GADPangleRTBNativeRenderer alloc] init];
-  [_nativeRenderer renderNativeAdForAdConfiguration:adConfiguration
-                                  completionHandler:completionHandler];
+                     completionHandler:(nonnull GADMediationNativeLoadCompletionHandler)completionHandler {
+    [GADMediationAdapterPangle setChildDirected:(adConfiguration.childDirectedTreatment
+                                             ? adConfiguration.childDirectedTreatment.integerValue
+                                             : -1)];
+    _nativeRenderer = [[GADPangleRTBNativeRenderer alloc] init];
+    [_nativeRenderer renderNativeAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 /// Set the COPPA setting in Pangle SDK.
 ///
-/// @param COPPA An integer value that indicates whether the app should be treated as
-/// child-directed for purposes of the COPPA.  0 means false. 1 means true. -1 means
-/// unspecified. Any value outside of -1, 0, or 1 will result in this method being a no-op.
-+ (void)setCOPPA:(NSInteger)COPPA {
-  if (COPPA != 0 && COPPA != 1 && COPPA != -1) {
+/// @param childDirected An integer value that indicates whether the app should be treated as
+/// child-directed for purposes of the COPPA.  See <a
+/// href="https://www.pangleglobal.com/integration/ios-initialize-pangle-sdk">
+/// Pangle's documentation</a> for more information about what values may be provided.
++ (void)setChildDirected:(NSInteger)childDirected {
+  if (childDirected != 0 && childDirected != 1 && childDirected != -1) {
     GADMPangleLog(@"Invalid COPPA value. Pangle SDK only accepts -1, 0 or 1.");
     return;
   }
-  if (BUAdSDKManager.initializationState == BUAdSDKInitializationStateReady) {
-    [BUAdSDKManager setCoppa:COPPA];
+  if (PAGSdk.initializationState == PAGSDKInitializationStateReady) {
+      PAGConfig.shareConfig.childDirected = childDirected;
   }
 }
-
-+ (void)setGDPR:(NSInteger)GDPR {
-  if (GDPR != 0 && GDPR != 1 && GDPR != -1) {
++ (void)GDPRConsent:(NSInteger)GDPRConsent {
+  if (GDPRConsent != 0 && GDPRConsent != 1 && GDPRConsent != -1) {
     GADMPangleLog(@"Invalid GDPR value. Pangle SDK only accepts -1, 0 or 1.");
     return;
   }
-  if (BUAdSDKManager.initializationState == BUAdSDKInitializationStateReady) {
-    [BUAdSDKManager setGDPR:GDPR];
+  if (PAGSdk.initializationState == PAGSDKInitializationStateReady) {
+      PAGConfig.shareConfig.GDPRConsent = GDPRConsent;
   }
-  _gdpr = GDPR;
+  _GDPRConsent = GDPRConsent;
 }
 
-+ (void)setCCPA:(NSInteger)CCPA {
-  if (CCPA != 0 && CCPA != 1 && CCPA != -1) {
++ (void)doNotSell:(NSInteger)doNotSell {
+  if (doNotSell != 0 && doNotSell != 1 && doNotSell != -1) {
```

Comment:
Only two values are supported, shouldn't be something like below?
```
  if (doNotSell != PAGGDPRConsentTypeNoConsent && doNotSell != PAGGDPRConsentTypeConsent) {
```

Reply:
We support three values. default(-1), noConsent(-1), consent(-1)